client/components/marketing-survey/cancel-jetpack-form/index.tsx

```diff
@@ -112,10 +147,27 @@ const CancelJetpackForm: React.FC< Props > = ( {
 			CANCEL_FLOW_TYPE.CANCEL_WITH_REFUND === flowType
 		) {
 			availableSteps.push( steps.CANCELLATION_REASON_STEP );
-		}
 
+			// During the cancellation decision, potentially show an offer
+			if (
+				provideCancellationOffer &&
+				cancellationOffer &&
+				isOfferPriceSameOrLowerThanPurchasePrice &&
+				offerDiscountBasedFromPurchasePrice >= 10
```

Comment:
We could think about simplifying these conditions. E.g. if `isOfferPriceSameOrLowerThanPurchasePrice` = true, it already implies that `cancellationOffer` is defined.

Reply:
Agreed that some of these conditions imply others being true as well, however I find this more readable in this case - I have a better idea of what all should be true/ defined by reading this list versus omitting implied values.